client/src/components/CompanyPage.jsx

```diff
@@ -3,12 +3,13 @@ import React from 'react';
 function CompanyPage(props) {
     let newCompany = props.companies.map((company) =>
         <ul className="flex-item">
-            <div className="company-name">{company.companyName}</div>
-            <div className="approval-rating"><i className="fa fa-tag"></i>{company.percentageApproved}%</div>
-            <div className="stock-price">${company.currentPrice}.99</div>
-            <div className="price-fluctuation">{company.percentageChange}%</div>
+            <li className="company-name">{company.companyName}</li>
+            <li className="approval-rating"><i className="fa fa-tag"></i><div className="tooltip">{company.percentageApproved}%<span className="tooltiptext">{company.percentageApproved}% of analysts agree {company.companyName} is a buy.</span></div></li>
+            <li className="stock-price">${company.currentPrice}.99</li>
```

Comment:
Should this decimal be hardcoded? Is it always .99 truly, or just hardcoding for now? consider randomizing to make it in sync with the goal of appearing like the app.